README.md

```diff
@@ -3,33 +3,161 @@
 
-* `moduleroot` templates get deployed on `new module`, `convert` and `update`; use them to enforce a common boilerplate for central files.
-* `moduleroot_init` templates get only deployed when the target module does not yet exist; use them to provide skeletons for files the developer needs to modify heavily.
-* `object_templates` templates are used by the various `new ...` commands for classes, defined types, etc.
+The following table shows the directories where different types of template are stored. 
 
-PDK builds the configuration of the module by reading a set of default configuration from `config_defaults.yml` and merging it with the contents of `.sync.yml` from module's root directory, if exists. Top-level keys of the resulting hash correspond to target files or represent a global configuration. Global configuration will be merged with the configuration hash for a particular target file. This allows a module developer to override/amend the configuration by putting new values into `.sync.yml`. A knockout prefix may be applied to elements within `.sync.yml` to [remove elements from the default set](#removing-default-configuration-values). Target files are created by rendering a corresponding template which is referring its configuration via the `@configs` hash. The data for a target file may include `delete: true` or `unmanaged: true` in order to have the particular file removed from the module or left unmanaged, respectively. The PDK can also be configured to make files executable if the file is executable in the template by including `manage_execute_permissions: true` in the data for the target file or globally via the `common` config key.
+|                   |               |
+| :-----------------|:--------------|
+| `moduleroot`      | Stores templates that are deployed on pdk new module, pdk convert and pdk update commands. Use the templates in this directory to enforce a common boilerplate for central files. |
+| `moduleroot_init` | Stores light versions of moduleroot templates. The templates in this directory create a skeleton for the files the developer will need. Deploy these templates when the target module does not yet exist. |
+| `object_templates`| Stores templates that are used by the various new <something> commands. These are in charge of building classes, defined types, etc |
+
+Use this README to understand the purpose, basic usage, and common configurable values for PDK templates. For further information about PDK, refer to our [documentation site](https://www.puppet.com/docs/pdk/2.x/pdk.html).
 
+#### Table of contents
+
+* [Introduction](#pdk-templates)
+* [How templates are used by PDK](#how-templates-are-used-by-pdk)
 * [Basic usage](#basic-usage)
-* [Values of config\_defaults](#values)
-* [Making local changes to the Template](#making-local-changes-to-the-template)
-* [Further Notes](#notes)
+* [Adjusting templates via /sync.yml](#adjusting-templates-via-syncyml)
```

Comment:
```suggestion
* [Adjusting templates via .sync.yml](#adjusting-templates-via-syncyml)
```
You could also keep the slash if it's meant to show it's at the root of the template repo, e.g. `/.sync.yml`

Reply:
That was probably a spelling mistake. Will amend to adhere to the standard throughout the document. Thanks for pointing this out.